besu/src/main/java/org/hyperledger/besu/services/TraceServiceImpl.java

```diff
@@ -191,7 +191,7 @@ private List<TransactionProcessingResult> trace(
     final ProtocolSpec protocolSpec = protocolSchedule.getByBlockHeader(block.getHeader());
     final MainnetTransactionProcessor transactionProcessor = protocolSpec.getTransactionProcessor();
     final BlockHeader header = block.getHeader();
-    tracer.traceStartBlock(block.getHeader(), block.getBody());
+    tracer.traceStartBlock(block.getHeader(), block.getBody(), block.getHeader().getCoinbase());
```

Comment:
You will need to use `miningBeneficiaryCalculator` here otherwise the `coinbase` will still be zero for Click